kyo-combinators/shared/src/main/scala/kyo/Constructors.scala

```diff
@@ -181,13 +167,13 @@ extension (kyoObject: Kyo.type)
     def traverse[A, S, S1](
         sequence: => Seq[A < S] < S1
     ): Seq[A] < (S & S1) =
-        sequence.flatMap((seq: Seq[A < S]) => Kyo.seq.collect(seq))
+        sequence.flatMap((seq: Seq[A < S]) => Kyo.collect(seq))
 
     def traverseDiscard[A, S, S1](
         sequence: => Seq[A < S] < S1
     ): Unit < (S & S1) =
         sequence.flatMap { (seq: Seq[A < S]) =>
-            Kyo.seq.collect(seq.map(_.map(_ => ()))).unit
+            Kyo.collect(seq.map(_.map(_ => ()))).unit
```

Comment:
is there a reason not to use `_.unit`?

Reply:
yeah, I've changed it to use `Kyo.collectDiscard`